config/software/libyaml-windows.rb

```diff
@@ -43,8 +43,8 @@
   command "7z.exe x #{project_file} -o#{tmpdir} -r -y", env: env
 
   # Now extract the files out of tar archive.
-  command "7z.exe x #{File.join(tmpdir, "libyaml-0.1.6-x86-windows.tar")} -o#{tmpdir} -r -y", env: env
+  command "7z.exe x #{File.join(tmpdir, 'libyaml-0.1.6-x86-windows.tar')} -o#{tmpdir} -r -y", env: env
 
   # Now copy over libyaml-0-2.dll to the build dir
-  copy "#{tmpdir}/bin/libyaml-0-2.dll", "#{install_dir}/embedded/bin/libyaml-0-2.dll"
+  copy '#{tmpdir}/bin/libyaml-0-2.dll', '#{install_dir}/embedded/bin/libyaml-0-2.dll'
```

Comment:
I wonder if we should just move to @fnichol's https://github.com/fnichol/finstyle

Test Kitchen is already using this much more sane set of Rubocop rules.